R/lsat_image.R

```diff
@@ -30,12 +30,17 @@ lsatGET <- function(url, dat, overwrite, ...) {
     message("File in cache")
     return(fpath)
   } else {
-    dir.create(dirname(fpath), showWarnings = FALSE, recursive = TRUE)
-    res <- GET(url, write_disk(path = fpath, overwrite = overwrite), ...)
-    # delete file if error, and stop message
+    temp_path = tempfile()
```

Comment:
if we went with `tempfile()` I think we'd need to make sure to close that connection on exit